test/flows/uk_benefits_abroad_flow_test.rb

```diff
@@ -0,0 +1,1328 @@
+                      which_benefit?: "jsa",
+                      which_country?: "liechtenstein",
+                      worked_in_eea_or_switzerland?: "no"
+      end
+
+      should "render the question" do
+        assert_rendered_question
+      end
+
+      context "next_node" do
+        should "have a next node of jsa_eea_going_abroad_maybe_outcome for a 'before_jan_2021' response if benefit is jsa" do
+          assert_next_node :jsa_eea_going_abroad_maybe_outcome, for_response: "before_jan_2021"
+        end
+
+        should "have a next node of wfp_going_abroad_eea_maybe_outcome for a 'before_jan_2021' response if benefit is winter_fuel_payment" do
+          add_responses which_benefit?: "winter_fuel_payment"
+          assert_next_node :wfp_going_abroad_eea_maybe_outcome, for_response: "before_jan_2021"
+        end
+
+        should "have a next node of esa_going_abroad_eea_outcome for a 'before_jan_2021' response if benefit is esa and going_abroad" do
+          add_responses which_benefit?: "esa",
+                        esa_how_long_abroad?: "esa_more_than_a_year"
+          assert_next_node :esa_going_abroad_eea_outcome, for_response: "before_jan_2021"
+        end
+
+        should "have a next node of esa_already_abroad_eea_outcome for a 'before_jan_2021' response if benefit is esa and already_abroad" do
+          add_responses going_or_already_abroad?: "already_abroad",
+                        which_benefit?: "esa",
+                        esa_how_long_abroad?: "esa_more_than_a_year"
+          assert_next_node :esa_already_abroad_eea_outcome, for_response: "before_jan_2021"
+        end
+
+        should "have a next node of db_going_abroad_eea_outcome for a 'before_jan_2021' response if benefit is disability_benefits and going_abroad" do
+          add_responses which_benefit?: "disability_benefits",
+                        db_how_long_abroad?: "permanent",
+                        db_claiming_benefits?: "yes"
+          assert_next_node :db_going_abroad_eea_outcome, for_response: "before_jan_2021"
+        end
+
+        should "have a next node of db_already_abroad_eea_outcome for a 'before_jan_2021' response if benefit is disability_benefits and already_abroad" do
+          add_responses going_or_already_abroad?: "already_abroad",
+                        which_benefit?: "disability_benefits",
+                        db_how_long_abroad?: "permanent",
+                        db_claiming_benefits?: "yes"
+          assert_next_node :db_already_abroad_eea_outcome, for_response: "before_jan_2021"
+        end
+
+        should "have a next node of jsa_not_entitled_outcome for a 'after_jan_2021' response if benefit is jsa" do
+          assert_next_node :jsa_not_entitled_outcome, for_response: "after_jan_2021"
+        end
+
+        should "have a next node of jsa_not_entitled_outcome for a 'no' response if benefit is jsa" do
+          assert_next_node :jsa_not_entitled_outcome, for_response: "no"
+        end
```

Comment:
It might be a bit neater to have a loop of `%w[after_jan_2021 no]` responses to save duplicating each of the 6 outcomes.